modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/internal/convert/WikiCreoleConvertProcess.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.wiki.web.internal.convert;
+package com.liferay.wiki.internal.convert;
```

Comment:
@JanisZhang Why these classes need to be moved together with `MediaWikiImporter`?